python-sdk/src/astro/databases/base.py

```diff
@@ -733,6 +733,9 @@ def get_sqlalchemy_template_table_identifier_and_parameter(
             self.get_table_qualified_name(table),
         )
 
+    def parameterize_variable(self, variable: str):
+        return ":" + variable
```

Comment:
Can you add a description of why this function is needed and does this need to be in BaseDatabase class?

If so, when would someone need to override it?